apis/sgv2-quarkus-test-common/src/test/resources/application.yaml

```diff
@@ -0,0 +1,12 @@
+# by default ignore bridge for data store props in tests
+stargate:
+  data-store:
+    ignore-bridge: true
```

Comment:
What does "ignore-bridge" do? Instead of calling bridge during tests it'll ... ?

Reply:
It controls whether we use the bridge's `GetSupportedFeatures` operation to find out if the persistence backend supports SAI, logged batches, etc. If false, this information must be provided via the configuration instead.

I couldn't find a reference in the Java code at first because it uses a different case: `DataStoreConfig.ignoreBridge()` 😡 Something to keep in mind when you try to find where config properties are used.

Then we use it in `DataStorePropertiesConfiguration`.

Reply:
Thanks, that makes sense. Name is bit vague but comment can in config help, and I am not sure what would be a better name ("access-metadata-via-bridge" is sort of long f.ex)

Reply:
I am fully up for name change of the config prop if you have a better idea?